src/transport/raw/Tuple.h

```diff
@@ -106,6 +106,8 @@ class Tuple : public Base
     {
         return TCPDisconnectImpl<0>(conn, shouldAbort);
     }
+
+    bool IsServerListenEnabled() override { return IsServerListenEnabledImpl<0>(); }
```

Comment:
Is this used?  It doesn't seem to be used.  The one use in Server gets something explicitly by index...

And what does this function even mean?  "We have some transport that's listening as a server"?

What does it even mean for a generic transport (e.g. UDP) to be "listening as a server"?